test/e2e/helpers/channel_single_event_helper.go

```diff
@@ -36,8 +36,7 @@ import (
 type SubscriptionVersion string
 
 const (
-	SubscriptionV1beta1 SubscriptionVersion = "v1beta1"
-	SubscriptionV1      SubscriptionVersion = "v1"
+	SubscriptionV1 SubscriptionVersion = "v1"
```

Comment:
since we have removed a bit of `V1` from other names - should we here too ? 

No biggie if not :-)  I just noticed that

Reply:
Yeah, ima change these in followup PRs, my opinion was that getting the API changes in and baked for almost a full release was more important, sounds like you're aok with that approach @matzew ? :)